io.github.jkozera.ZevDocs.json

```diff
@@ -0,0 +1,74 @@
+{
+    "app-id" : "io.github.jkozera.ZevDocs",
+    "runtime" : "org.gnome.Sdk",
+    "runtime-version" : "3.28",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.golang"
+    ],
+    "command" : "zevdocs",
+    "rename-icon": "zevdocs",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--device=dri",
+        /* Needs to talk to the network: */
+        "--share=network",
+        "--filesystem=host:ro",
+        "--filesystem=xdg-run/dconf",
+        "--filesystem=~/.config/dconf:ro",
+        "--talk-name=ca.desrt.dconf",
+        "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
+    ],
+    "build-options" : {
+        "cflags" : "-O2",
+        "cxxflags" : "-O2",
+        "env" : {
+            "V" : "1"
+        },
+        "build-args": ["--share=network"]
+    },
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/share/pkgconfig",
+        "/share/aclocal",
+        "/man",
+        "/share/man",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name" : "zevdocs",
+            "config-opts" : [
+                "--enable-flatpak-build"
+            ],
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "ssh://git@github.com/jkozera/zevdocs.git",
```

Comment:
Appdata misses releases and [OARS](https://hughsie.github.io/oars/) sections.

Reply:
Added.